locales/tr-TR.yml

```diff
@@ -0,0 +1,239 @@
+hexpatch: HexPatch
+
+errors:
+  enable_raw_mode: "Ham mod etkinleştirilemedi"
+  setup_commands: "Kurulum komutları çalıştırılamadı"
+  create_terminal: "Terminal oluşturulamadı"
+  clear_terminal: "Terminal temizlenemedi"
+  create_app: "Uygulama oluşturulamadı"
+  disable_raw_mode: "Ham mod devre dışı bırakılamadı"
+  teardown_commands: "Kapatma komutları çalıştırılamadı"
+  show_cursor: "İmleç gösterilemedi"
+  detect_terminal_theme: "Terminal teması algılanamadı: %{e}"
+  load_settings: "Ayarlar yüklenirken hata oluştu: %{e}"
+  connect_ssh: "%{ssh} adresine bağlanılamadı: %{e}"
+  load_plugins: "Eklentiler yüklenirken hata oluştu: %{e}"
+  create_comments: "Yorumlar oluşturulamadı: %{e}"
+  write_comments: "Yorumlar yazılamadı: %{e}"
+  parse_comments: "Yorumlar ayrıştırılamadı: %{e}"
+  read_comments: "Yorumlar okunamadı: %{e}"
+  no_symbols_found: "Sembol bulunamadı"
+  no_matching_symbols: "Arama desenine uyan sembol bulunamadı"
+  no_matching_comments: "Arama desenine uyan yorum bulunamadı"
+  invalid_address: "Geçersiz adres: %{address}"
+  invalid_virtual_address: "Geçersiz sanal adres: %{address}"
+  symbol_not_found: "Sembol bulunamadı: %{symbol}"
+  instruction_no_mnemonic: "Mnemonic alınamadı"
+  instruction_no_operands: "Operandlar alınamadı"
+  virtual_address_not_found: "Sanal adres %{address} bulunamadı"
+  cursor_position: "Kaydırma, imlecin görünür olması için yeterli olmalıdır"
+  create_decoder: "Çözücü oluşturulamadı"
```

Comment:
Word "çözücü" might require disambiguation between "codec" and "decoder", maybe just leave it as "decoder"; unless the program gives enough context. This may be ok as-is, I'm just marking stuff to make a review as thorough as possible @orhun